src/main/java/miles/command/MarkCommand.java

```diff
@@ -0,0 +1,30 @@
+package miles.command;
+
+import miles.Ui;
+import miles.Storage;
+import miles.TaskList;
+
+public class MarkCommand extends Command {
+    private String input;
+
+    public MarkCommand(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Return task number from user input.
```

Comment:
Just a minor problem, JavaDoc comments should start with Returns ... 
and include punctuation behind each parameter description.
Mistake is repeated in other files too, remember to keep a look out!